x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/evaluate_panel.tsx

```diff
@@ -7,6 +7,7 @@
 import React, { FC, useState, useEffect, Fragment } from 'react';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
+import { IndexPattern } from 'ui/index_patterns';
```

Comment:
Looking at `kibana_context.ts` can this import be from `src/plugins/data/public` - `IndexPatternsContract` ?

Reply:
I believe the type should be `IIndexPattern` from the same location.
I plan to change all of our uses of `IndexPattern` to be of that type in future NP work.

Reply:
Updated 0a79e2a39475beff24b51a40e78f6d6d3e806cca